woocommerce/woocommerce-android
# Activate: Add "What is WordPress.com" button on "No WPCom account found" screen

Part of: #7121


### Description


This adds a "What is WordPress.com?" button on the "No WPCom account found" screen.

It might appear to be redundant since the button also exists on the email login screen before this, but it might be worth showing it again as it appears on an error screen (so hopefully be more timely and helpful). We'll keep track and see how useful it is in practice.

(Tracking will be added in a separate PR)

### Testing instructions


1. Start with login flow "Continue with WordPress.com",
2. Enter a random, invalid address (`asdadaer@adsaae.com`),
3. On the "No WPCcom account found" screen that opens, make sure "What is WordPress.com" is shown.
4. Tap it and make sure it loads webview correctly.

### Images/gif


<img src="https://user-images.githubusercontent.com/266376/183638010-50befd20-59e0-4521-90bc-a07df12afe5f.png" width="42%" />
